e2e/provision/playbooks/roles/kpt/molecule/default/prepare.yml

```diff
@@ -10,21 +10,6 @@
 
 - name: Boostrap servers
   hosts: all
-  pre_tasks:
-    - name: Update Apt cache
-      ansible.builtin.raw: apt-get update --allow-releaseinfo-change
-      become: true
-      changed_when: false
-    - name: Install pip package
-      become: true
-      ansible.builtin.package:
-        name: python3-pip
-        state: present
-      when: ansible_distribution == 'Ubuntu'
-    - name: Install kubernetes python package
-      become: true
-      ansible.builtin.pip:
-        name: kubernetes==26.1.0
```

Comment:
Would be cool to pull this out to variables/defaults file otherwise it will get forgotten. Otherwise approved

Reply:
This playbook is used for the integration tests of the Ansible roles, so it doesn't pretend to be consumed by the end user.